chpl/chpl-service/src/main/java/gov/healthit/chpl/caching/CacheNames.java

```diff
@@ -25,7 +25,7 @@ private CacheNames() {}
     public static final String FIND_SURVEILLANCE_RESULT_TYPE = "findSurveillanceResultType";
     public static final String TEST_PROCEDURES = "testProcedures";
     public static final String TEST_DATA = "testData";
-    public static final String TEST_FUNCTIONALITY_MAPS = "testFunctionalityMaps";
+    public static final String FUNCTIONALITY_TESTED_MAPS = "testFunctionalityMaps";
```

Comment:
Should the name be changed too?

Reply:
Yes but I didn't want to because I have to change it in every environment and it hardly seems worth it. Updated it though.